packages/trezor-user-env-link/src/websocket-client.ts

```diff
@@ -268,7 +268,7 @@ export class WebsocketClient extends TypedEmitter<WebsocketClientEvents> {
     }
 
     waitForTrezorUserEnv() {
-        return new Promise<void>(async (resolve, reject) => {
+        return async () => {
```

Comment:
Thank you, `regtest.ts` looks perfect, but here you accidentally changed it to HOF.
I meant just this, without the `return async () => {`:

```
    async waitForTrezorUserEnv() {
        // unfortunately, it can take incredibly long for trezor-user-env to start, we should
        // do something about it
        const limit = 300;
        let error = '';

        console.log('waiting for trezor-user-env');

        for (let i = 0; i < limit; i++) {
            if (i === limit - 1) {
                console.log(`cant connect to trezor-user-env: ${error}\n`);
            }
            await delay(1000);

            try {
                const res = await fetch(USER_ENV_URL.DASHBOARD);
                if (res.status === 200) {
                    console.log('trezor-user-env is online');

                    return;
                }
            } catch (err) {
                error = err.message;
                // using process.stdout.write instead of console.log since the latter always prints also newline
                // but in karma, this code runs in browser and process is not available.
                if (typeof process !== 'undefined') {
                    process.stdout.write('.');
                } else {
                    console.log('.');
                }
            }
        }

        throw error;
    }
```